envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/resource/listeners/filters/RBACFilterPermissions.kt

```diff
@@ -3,11 +3,15 @@ package pl.allegro.tech.servicemesh.envoycontrol.snapshot.resource.listeners.fil
 import io.envoyproxy.envoy.api.v2.route.HeaderMatcher
 import io.envoyproxy.envoy.config.rbac.v2.Permission
 import io.envoyproxy.envoy.type.matcher.PathMatcher
+import io.envoyproxy.envoy.type.matcher.RegexMatcher
 import io.envoyproxy.envoy.type.matcher.StringMatcher
 import pl.allegro.tech.servicemesh.envoycontrol.groups.IncomingEndpoint
 import pl.allegro.tech.servicemesh.envoycontrol.groups.PathMatchingType
 
 class RBACFilterPermissions {
+
+    private val paramRegex = Regex("\\{\\w+\\}")
```

Comment:
I think it might not include all characters allowed in a path. https://tools.ietf.org/html/rfc2396

Reply:
I've added all "unreserved characters" allowed in URI segments

Reply:
It seems like RFC2396 is updated by 3986 (https://www.ietf.org/rfc/rfc3986.txt), and characters change so I will update it soon